platform/node/test/js/map.test.js

```diff
@@ -432,12 +428,9 @@ test('Map', function(t) {
         t.test('requires a style to be set', function(t) {
             var map = new mbgl.Map(options);
 
-            t.throws(function() {
-                map.render({}, function() {});
-            }, /Style is not loaded/);
+            map.render({}, function(err) { if (err) t.end(); });
```

Comment:
Should assert on the error here instead of making `t.end()` conditional, which will cause the tests to hang instead of fail and continue if no error is returned.